src/outfit.ts

```diff
@@ -205,6 +214,23 @@ export function meatOutfit(embezzlerUp: boolean, requirement?: Requirement, sea?
       forceEquip.push($item`Kramco Sausage-o-Matic™`);
     }
   }
+
+  if (embezzlerUp) {
+    const currentWeapon = 25 * findLeprechaunMultiplier(meatFamiliar());
+    const value = ((75 - currentWeapon) * (750 + baseMeat)) / 100;
```

Comment:
```suggestion
    const currentWeapon = equippedItem($slot`weapon`);
    const currentWeaponValue =
      ((numericModifier(currentWeapon, "Meat Drop") +
        2.5 *
          numericModifier(currentWeapon, "Familiar Weight") *
          findLeprechaunMultiplier(meatFamiliar())) *
        (750 + baseMeat)) /
      100;
    const value = 75 * (750 + baseMeat)) / 100 - currentWeaponValue;
```

Reply:
We can by no means guarantee that the weapon currently occupying that slot is a weapon that is part of our meatoutfit.

Reply:
I guess we could record our meatOutfit weapon as a global variable (or store it as a garbo mafia preference)?

Reply:
I feel like that's also a pretty bad solution for a variety of reasons (we don't have a single meatOutfit, or even a single _barf_ meatOutfit), but I'd welcome a more in-depth approximation than 25 * lepMult (which assumes you're either using a garbage sticker or a cursed pirate cutlass, basically)

Reply:
My idea was to constantly track and update our meatWeapon every barf turn (after gearing, and right before we run an adventure).

But, disregarding opportunity costs, as long as the sticker sword is profitable I don't think the inefficiencies here (which only applies to certain circumstances and not everyone) would be too big of a deal.

Reply:
I'm going to go ahead and declare this out of scope and we can try to improve it later on. But I don't particularly like any solution and I think this is Fine